thorium/response.py

```diff
@@ -71,11 +71,15 @@ def __init__(self, *args, **kwargs):
 
     def get_response_data(self):
         data = []
-        if self.resources:
+        if getattr(self, 'resources', False):
+            data = []
             self._sort()
             self._paginate()
             for res in self.resources:
                 data.append(OrderedDict(res.sorted_items()))
+        if getattr(self, 'resource', False):
+            data = None
+            data = OrderedDict(self.resource.sorted_items())
```

Comment:
I think Devon means the `data = None`, line 81, which doesn't do anything because data is immediately overwritten on line 82.